src/pages/events.tsx

```diff
@@ -0,0 +1,176 @@
+              filterState === filterStates.all ? "!bg-events-baby-blue" : ""
+            }`}
+            onClick={() => handleUiClick(filterStates.all)}
+          >
+            <ListIcon height={20} width={20} className={`${filterState === filterStates.all ? "text-events-user-card-bg fill-current" : ""}`} />{" "}
+            <p className={`${filterState === filterStates.all ? "text-events-user-card-bg" : ""} text-nowrap`}>All Events</p>
+          </div>
+          <div
+            className={`bg-events-card-bg p-2 h-[46px] rounded-lg flex-row justify-center items-center space-x-1 px-20 shrink hidden lg:flex cursor-pointer ${
+              filterState === filterStates.saved ? "!bg-events-baby-blue" : ""
+            }`}
+            onClick={() => handleUiClick(filterStates.saved)}
+          >
+            <Bookmark height={20} width={20} className={`${filterState === filterStates.saved ? "text-events-user-card-bg fill-current" : ""}`} />
+            <p className={`${filterState === filterStates.saved ? "text-events-user-card-bg" : ""}`}>Saved</p>
+          </div>
+        </div>
+        <div className="flex flex-row space-x-3 mb-6 lg:hidden">
+          <div
+            className={`bg-events-card-bg p-2 h-[46px] rounded-lg flex-row justify-center items-center space-x-1 shrink flex grow lg:hidden cursor-pointer ${
+              filterState === filterStates.all ? "!bg-events-baby-blue" : ""
+            }`}
+            onClick={() => handleUiClick(filterStates.all)}
+          >
+            <ListIcon height={20} width={20} className={`${filterState === filterStates.all ? "text-events-user-card-bg fill-current" : ""}`} />
+            <p className={`${filterState === filterStates.all ? "text-events-user-card-bg" : ""} text-nowrap`}>All Events</p>
+          </div>
+          <div
+            className={`bg-events-card-bg p-2 h-[46px] rounded-lg flex-row justify-center items-center space-x-1 shrink flex grow lg:hidden cursor-pointer ${
+              filterState === filterStates.saved ? "!bg-events-baby-blue" : ""
+            }`}
+            onClick={() => handleUiClick(filterStates.saved)}
+          >
+            <Bookmark height={20} width={20} className={`${filterState === filterStates.saved ? "text-events-user-card-bg fill-current" : ""}`} />
+            <p className={`${filterState === filterStates.saved ? "text-events-user-card-bg" : ""}`}>Saved</p>
+          </div>
+        </div>
+
+        <EventDashboard events={displayedEvents} user={email} saved={saved} registered={registered} setSaved={setSaved} />
+      </div>
+    </main>
+  );
+}
+
+export async function getStaticProps() {
+  let events = await fetchBackend({ endpoint: "/events", method: "GET", authenticatedCall: false });
+
+  events.sort((a: BiztechEvent, b: BiztechEvent) => {
+    return new Date(b.startDate).getTime() - new Date(a.startDate).getTime();
+  });
+
+  events = events.filter((e: BiztechEvent) => {
+    return e.isPublished;
+  });
```

Comment:
Not particular to this PR, but note for @AllanT102 - in the long-term, we should be change our backend API to only expose published events for non-admins.